### ISSUE TITLE
Armory in offline mode - no clear way to bring it "online"

### ISSUE THREAD

**Issue by fresheneesz** [etotheipi/BitcoinArmory#232]
I just installed Armory 0.91.2-beta and my computer's nice and online. Its downloading the blockchain right now in fact. But when I click on "send bitcoins" it tells me that Armory is in offline mode. It doesn't tell me anything about how to switch to "online mode". My computer's online, Armory told me I shouldn't be running QT separately, so I'm not. So what gives?

**Comment by dialex** [comment]
Two years have passed and still there's no intuitive way for the user to know how to go online.

> Until the blockchain is downloaded, you can't send bitcoins.

And I can't see the balance of my imported wallet either...
Is that done in the background? How can I know the progress of that? How long do I have to wait?

**Comment by fresheneesz** [comment]
I'm reopening this because Armory continues to be a goddamn pain in the ass. I FINALLY downloaded the blockchain using bitcoin core after doing some symlink tricks to speed up the process, and now Armory just looks at me like a cow looks at an oncoming train saying "offline". Its embarrassing that these usability issues still exist.

**Comment by d4rkd0s** [comment]
I have the same issue, my bitcoin-core has downloaded everything and it online/uptodate. Yet amory, states that I am Offline... ?

However, @fresheneesz I was able to get it working! I had to select both my bitcoin install dir and my blockchain location, and then it worked perfectly!

![armory settings](https://i.imgur.com/R4nuEn3.png)

@fresheneesz try it out... it'll start parsing your downloaded blocks and go green!

**Comment by fresheneesz** [comment]
Thanks for the note! I've actually stopped using armory because of the years of usability issues I've had with it. Switched to electrum and its a lot easier to use. 

They really need to have a third option for separating the database directory from the blockchain storage directory, so people can put the database on their SSD and keep the blockchain itself on a larger HDD - they could use the same symlink tricks I used manually.
